python/lsst/sconsUtils/dependencies.py

```diff
@@ -622,7 +622,16 @@ def _tryImport(self, name):
             filename = os.path.join(path, name + ".cfg")
             if os.path.exists(filename):
                 try:
-                    module = imp.load_source(name + "_cfg", filename)
+                    # https://docs.python.org/3/library/importlib.html#importing-a-source-file-directly
+                    # Need to specify SourceFileLoader since the files do not
+                    # have a .py extension.
+                    module_name = f"{name}_cfg"
+                    loader = importlib.machinery.SourceFileLoader(module_name, filename)
+                    spec = importlib.util.spec_from_file_location(module_name, filename,
+                                                                  submodule_search_locations=None,
+                                                                  loader=loader)
+                    module = importlib.util.module_from_spec(spec)
+                    spec.loader.exec_module(module)
```

Comment:
FWIW, I'm not sure why I used `imp` here in the first place; it's certainly conceivable that something like this would also work:
```
with open(filename, "r") as buffer:
    vars = {}
    exec(buffer.read(), vars)
module = SimpleNamespace(**vars)  # not really a module, but close enough
```

Reply:
Incredibly the way I'm doing it seems to be the documented approach. I couldn't believe it myself. I'm sure you used `imp` because it was the default old way to do it.